src/scouting/public/js/match-scouting.js

```diff
@@ -66,27 +66,29 @@ var previousTimer = [];
 
     undo: (button) => {
       button.element.addEventListener("click", () => {
-        const undoneId = actionQueue.pop().id; //remove the last action from the action queue
-        const undoneButton = buttons.find((x) => x.id === undoneId);
-
-        //special case for match-control buttons which have extra undo funcitonality without executables
-        if (undoneButton.type === "match-control") {
-          time = matchScoutingConfig.timing.totalTime; //reset timer
-          ScoutingSync.updateState({
-            status: ScoutingSync.SCOUTER_STATUS.WAITING,
-          }); //tell the server that you are now waiting to start
-          clearInterval(undoneButton.timerInterval); //clear the timing interval
-          undoneButton.element.innerText = "Start Match";
-          timerActive = false;
-          showLayer(0);
-        }
+        if (actionQueue.length > 2) {
```

Comment:
Let's replace the literal 2 with a value in matchScoutingConfig. Something like minimumActionsForUndo. This will make it easier to maintain in the future as different games may require different values.

Reply:
Perhaps under variables in match-scouting.json?